snuba/writer.py

```diff
@@ -10,6 +11,8 @@
 
 logger = logging.getLogger('snuba.writer')
 
+TableRow = Mapping[str, Any]
```

Comment:
I like this but it also seems a little confusing to manage both `snuba.writer.TableRow` and `snuba.snapshots.TableRow`.

Reply:
I think I can find a better name, but I would not conflate the two because it is a coincidence that they have the same type now. A binary importer would have a different TableRow type while the writer one will still be Mapping[str, Any]